### PR TITLE
[SSL] Clean up

### PR DESCRIPTION
**Pull request by kodster28**
Cleaning up SSL docs for content strategy, but not including Keyless SSL or client certs in this PR.

### CODE REVIEW

File: products/ssl/src/content/edge-certificates/advanced-certificate-manager/manage-certificates.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+1. Log into your Cloudflare account and select a domain.
+1. Select **SSL/TLS** > **Edge Certificates**.
+1. Select **Order Advanced Certificate**.
+1. If Cloudflare does not have your billing information, you will need to enter that information.
+1. Enter the following information:
+    - Certificate Authority
+    - Certificate Hostnames
+    - Validation method
+    - Certificate Validity Period
+1. Select **Save**.
```
Comment (abracchi-tw) [inline products/ssl/src/content/edge-certificates/advanced-certificate-manager/manage-certificates.md]: don't remember what we decided here, but when buttons are involved I always use `click` (as in: you're not selecting from different options)

File: products/ssl/src/content/edge-certificates/advanced-certificate-manager/manage-certificates.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+<Aside type='warning' header='Important'>
+
+Selecting Let’s Encrypt as a CA limits a certificate to txt validation_method, 90 validity_days, omission of cloudflare_branding, and 2 host entries (one for the zone name and one for the subdomain wildcard of the zone name, such as `example.com`, `*.example.com`).
```
Comment (abracchi-tw) [inline products/ssl/src/content/edge-certificates/advanced-certificate-manager/manage-certificates.md]: should `Let's Encrypt` be in bold?

File: products/ssl/src/content/edge-certificates/advanced-certificate-manager/manage-certificates.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+### Using the API
+
+To delete a certificate, send a [DELETE request](https://api.cloudflare.com/#certificate-packs-delete-advanced-certificate-manager-certificate-pack) to the Cloudflare API.
+
+---
+
+## Restart validation
+
+To restart validation for a certificate in a `validation_timed_out` status, sending a [PATCH request](https://api.cloudflare.com/#certificate-packs-restart-validation-for-advanced-certificate-manager-certificate-pack) to the API.
```
Comment (abracchi-tw) [inline products/ssl/src/content/edge-certificates/advanced-certificate-manager/manage-certificates.md]:
```suggestion
To restart validation for a certificate in a `validation_timed_out` status, send a [PATCH request](https://api.cloudflare.com/#certificate-packs-restart-validation-for-advanced-certificate-manager-certificate-pack) to the API.
```

File: products/ssl/src/content/ssl-tls/cipher-suites.md [hunk 4]
```diff
@@ -23,37 +29,40 @@ ECDHE-RSA-AES128-SHA256|❌|❌|✅|❌
...
 
 --------
 
-## Matching on origin
+## Matching on origin (optional)
+
+Cloudflare will [present the cipher suites to your origin](/origin-configuration/cipher-suites/) and your server will select whichever cipher suite it prefers.
 
-If you would like to support the same cipher suites on your origin that Cloudflare supports at our edge, the nginx configuration below can be used. If you are terminating TLS on your origin using something other than nginx, please refer to that application’s documentation.
+However, if you wanted to ensure that your origin server supports the same cipher suites that Cloudflare supports at our edge, use the following nginx configuration. If you are terminating TLS on your origin using something other than nginx, please refer to that application’s documentation.
```
Comment (abracchi-tw) [inline products/ssl/src/content/ssl-tls/cipher-suites.md]:
```suggestion
However, if you want to ensure that your origin server supports the same cipher suites that Cloudflare supports at our edge, use the following nginx configuration. If you are terminating TLS on your origin using a different method, please refer to that application’s documentation.
```

Comment (abracchi-tw) [reply products/ssl/src/content/ssl-tls/cipher-suites.md]: feel free to disregard my suggestion but `something` felt a little too vague

**Review by abracchi-tw — APPROVED** [review]

File: products/ssl/src/content/ssl-tls/cipher-suites.md (hunk 4 above)
Comment (pedrosousa) [reply products/ssl/src/content/ssl-tls/cipher-suites.md]: Nit: I'd remove "please".

Comment (pedrosousa) [reply products/ssl/src/content/ssl-tls/cipher-suites.md]: Nit2: The product's official name is NGINX (all caps).

**Pull request merged**